docs/root/development/performance/device_conditions.rst

```diff
@@ -0,0 +1,73 @@
+.. _device_conditions:
+
+Analysis of device conditions
+=============================
+
+Results
+~~~~~~~
+
+iOS
+---
+
+* TODO
+
+Android
+-------
+
+We did not observe any issues when switching between background/foreground and between WiFi/cellular. Network requests
+resumed and were successful after a brief period of time.
+
+Experimentation method
+~~~~~~~~~~~~~~~~~~~~~~
+
+iOS
+---
+
+* TODO
+
+Android
+-------
+
+Getting the build:
+
+1. Build the library using ``bazel build android_dist --config=android``
+2. Control: ``bazel mobile-install //examples/kotlin/control:hello_control_kt``
+3. Envoy: ``bazel mobile-install //examples/kotlin/hello_world:hello_envoy_kt --fat_apk_cpu=armeabi-v7a``
+
+Lifecycle experiment steps:
+
+1. Open the example application
+2. Background the application
```

Comment:
Out of curiosity, what is behavior during backgrounding? Requests just stop happening?

Reply:
When it's unplugged, the application no longer makes requests